### PR TITLE
Added manual dispatch

### PR DESCRIPTION
**Pull request by zvonkok**
Ignore CI runs on README
Enable manual workflow dispatch

### CODE REVIEW

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     paths-ignore:
       - "**/README.md"
+  workflow_dispatch:
```
Comment (ArangoGutierrez) [inline .github/workflows/image.yaml:7]: Don't we need a bit more after `workflow_distpatch` eg https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

**Review by ArangoGutierrez — COMMENTED** [review]
Looks good, just  a simple question

File: .github/workflows/image.yaml (hunk 1 above)
Comment (zvonkok) [reply .github/workflows/image.yaml:7]: Maybe later but I do not see why we need any args for now.

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

File: .github/workflows/image.yaml (hunk 1 above)
Comment (ArangoGutierrez) [reply .github/workflows/image.yaml:7]: ok let's call it a TODO

**Pull request merged**